packages/govuk-frontend/src/govuk/components/header/header.test.js

```diff
@@ -221,6 +221,30 @@ describe('Header navigation', () => {
           message: 'Header: $module is not an instance of "HTMLElement"'
         })
       })
+
+      it('does not throw if the toggle is absent', async () => {
+        // The default example is rendered without navigation
+        // and should keep rendering. No expectations as the JavaScript
+        // will just return early. All we ask of that test is for it not
+        // to throw during the initialisation
+        await renderAndInitialise(page, 'header', {
+          params: examples.default
+        })
+      })
+
+      it('throws if the toggle is present, but no menu is there', async () => {
```

Comment:
Only a minor one, but can you pick your favourite wording across the PRs?

Helps when searching the project for "is not set" or "throws when" etc

### Suggestion from [#4104](https://github.com/alphagov/govuk-frontend/pull/4104/files#diff-4579399f94ca47997fb8c34cf74e61fe87a1ca2ffadb5d6050302ae285af6112R342)

```suggestion
      it('throws when #navigation is not set', async () => {
```

<br>

### Suggestion from [#4199](https://github.com/alphagov/govuk-frontend/pull/4199/files#diff-24879075b6d7335064bd2d9c2919edcb111443bf3f52549b048e4a16cbd9dde4R243)

```suggestion
      it('throws when the navigation is missing', async () => {
```

Reply:
Happy to settle for `throws when ... is missing`. I still want to complement that last one, as it'll only throw if it's missing *and* there is a toggle in the markup.

Reply:
You choose 😊